src/base.config.ts

```diff
@@ -190,7 +174,12 @@ export default function webpackConfigFactory(args: any): webpack.Configuration {
 						getUMDCompatLoader({ bundles: args.bundles }),
 						{
 							loader: 'ts-loader',
-							options: { onlyCompileBundledFiles: true, instance: jsonpIdent, transpileOnly: true, compilerOptions }
+							options: {
+								onlyCompileBundledFiles: false,
```

Comment:
Can we leave this as `true` until we make the changes for the custom element transformer?

Reply:
Done!